### PR TITLE
add output name and id to view_info

### PR DESCRIPTION
**Pull request by marcusbritanicus**

### CODE REVIEW

File: proto/wayfire-information.xml [hunk 1]
```diff
@@ -53,7 +53,7 @@
       <description summary="Export information about a view to a client">
 	Provide client with information about a view.
       </description>
-      <arg name="view_id" type="int" summary="view wayfire ID"/>
+      <arg name="view_id" type="uint" summary="view wayfire ID"/>
```
Comment (soreau) [inline proto/wayfire-information.xml:56]: This change should be in a separate patch.

Comment (marcusbritanicus) [reply proto/wayfire-information.xml]: Done!

**Comment by soreau** [comment]
I also have this notion that if we are exposing the output id, maybe we should also show the name of the output?

**Comment by marcusbritanicus** [comment]
soreau, I've added the output name as well. Do have a look.

**Comment by soreau** [comment]
LGTM, Thanks!
